packages/react-menu/src/MenuGroupDivider/renderMenuGroupDivider.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { getSlots } from '@fluentui/react-utils';
+import { MenuGroupDividerState } from './MenuGroupDivider.types';
+
+/** Function that renders the final JSX of the component  */
```

Comment:
**suggestion:** can we be consistent with JSDoc format (multiline all the time) ? 

--> 

```
/**
 * Lorem ipsum
 */
```